corehq/apps/translations/app_translations.py

```diff
@@ -724,23 +724,36 @@ def update_form_translations(sheet, rows, missing_cols, app):
         # Tell the user this?
         return msgs
 
-    try:
-        itext = xform.itext_node
-    except XFormException:
-        return msgs
+    def find_itext_node(xform):
+        # find the bucket node that holds all translations.
+        # it has a bunch of nodes, one for each lang, which then
+        # has translations for all labels as a child node, example
+        # <itext>
+        #   <translation lang="en" default="">
+        #     <text id="name-label">
+        #       <value>Name2</value>
+        #       <value form="image">image_path</value>
+        #     </text>
+        #   </translation>
+        # </itext>
+        return xform.itext_node
+
+    itext = find_itext_node(xform)
```

Comment:
Are you intentionally not catching XFormException?

Reply:
https://github.com/dimagi/commcare-hq/pull/24075#discussion_r276968475

Reply:
(Sorry. I missed the earlier conversation about this.)

Reply:
No worries :)